### PR TITLE
Add interface option to enable/disable the web terminal 

### LINKED ISSUE #5014: Add to admin/interface option to enable/disable the web terminal
This will use the new feature gate implemented in the backend.

**Notes:**
- the checkbox will be enabled only if "Enable OIDC Kubeconfig" is checked, as it's a prerequisite for using the web terminal

- Button "Open Terminal", in cluster page, should be displayed only if both "Enable OIDC Kubeconfig" and "Enable Terminal" are checked

### PR DESCRIPTION
**Pull request by KhizerRehan**
Signed-off-by: khizerrehan <khizerrehan92@gmail.com>

**What this PR does / why we need it**:

This PR adds new interface option to enable/disable the web terminal from cluster details page

https://user-images.githubusercontent.com/17727069/199467727-1d41b566-05e5-4c85-8e4d-2dfb10fbaffa.mp4

**Which issue(s) this PR fixes**:

Fixes #5014 

/kind feature
/kind design
/kind chore

**What type of PR is this?**

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Adds new option under admin/interface page to enable/disable web terminal from cluster details page.
```

**Documentation**:

```documentation
TBD
```

### CODE REVIEW

File: src/app/cluster/details/cluster/template.html [hunk 1]
```diff
@@ -68,7 +68,8 @@
       <span>Open Dashboard</span>
     </a>
 
-    <button *ngIf="(settingsService.adminSettings | async)?.enableOIDCKubeconfig"
+    <button *ngIf="(settingsService.adminSettings | async)?.enableOIDCKubeconfig &&
```
Comment (Waseem826) [inline src/app/cluster/details/cluster/template.html:71]: I think its better if we move subscription to `ts` file and create a class property for `adminSettings`.

**Review by Waseem826 — CHANGES_REQUESTED** [review]
Some feedback. PTAL

File: src/app/settings/admin/interface/template.html [hunk 2]
```diff
@@ -109,6 +109,23 @@
...
       </div>
+
+
+      <div fxLayout="row">
+        <div fxFlex="16%"
+             fxLayoutAlign=" center"
+             class="entry-label">
+          <span>Enable WebTerminal</span>
+          <div class="km-icon-info km-pointer"
+               matTooltip="Show/Hide &quot;Web Terminal&quot; button on cluster details and allow/block web terminal pod access thorough terminal."></div>
```
[the remark this answers is not in the document]
Comment (KhizerRehan) [reply src/app/settings/admin/interface/template.html:120]: yes!

**Review by Waseem826 — APPROVED** [review]
/hold 
Wait for unit tests to pass

**Review by Waseem826 — APPROVED** [review]

**Pull request merged**